mod-calendar/src/main/resources/bama/mod-calendar/features/calendarPeriods.feature

```diff
@@ -202,7 +211,7 @@ Feature: Calendar periods
     Given path 'calendar/periods/' + servicePointId + '/period/' + periodId
     When method GET
     Then status 404
-    And match $.errors[0].message == 'Openings with id \'' + periodId + '\' is not found'
```

Comment:
Error messages were revised (and are now localized, however, these tests do not include that).